src/lsdb/dask/crossmatch_catalog_data.py

```diff
@@ -166,6 +171,57 @@ def crossmatch_catalog_data(
     return ddf, partition_map, alignment
 
 
+def generate_meta_df_for_joined_tables(
+        catalogs: Sequence[Catalog],
+        suffixes: Sequence[str],
```

Comment:
So I was using tuple for suffixes everywhere because I thought that's a good way to represent it. And I think for a specific method like join or crossmatch where with a Tuple you can specify that it should contain exactly two strings it's the best choice. But when I refactored this into a function I figured it could be more general and work for any number of catalogs and suffixes, and so sequence is more appropriate so lists could also be used, as long as catalogs and suffixes are the same length which is checked for.